src/aap_eda/api/serializers/activation.py

```diff
@@ -92,6 +95,11 @@ def to_representation(self, activation):
             activation.ruleset_stats
         )
 
+        sources = []
+        if activation.sources:
+            for source in activation.sources.all():
```

Comment:
Is there a specific reason to avoid a list comprehension?  E.g.,

> sources = [
    SourceOutSerializer(source).data for source in activation.sources.all()
]

Reply:
@jshimkus-rh updated to use list comprehension

Reply:
@mkanoor Want to be certain...
If it's not necessary, great; but I assumed the presence of `if activation.sources:` in the previous commit was necessary before using that value.

Reply:
@jshimkus-rh I checked it locally in my testing the check is not needed since the ManyToMany Manager that sits in between the 2 tables returns an empty array if it doesn't have a relationship